### ISSUE TITLE
Scrape posts before downloading

### ISSUE THREAD

**Issue by bezaaas** [sim0n00ps/OF-DL#417]
Is it possible to scrape posts first and then start downloading

**Comment by bezaaas** [comment]
Maybe I was misunderstood. I meant that the downloading would not start until all the posts scraped (for subscribed/choiced models).

**Comment by melithine** [comment]
Right, that would require some extensive work, I believe. Right now it scrapes a model's content of XYZ type, then downloads, then moves on to the next content type, then the next model. Changing that behavior would not be simple. The exception is the purchased tab scraper which works differently (due to its nature).

I don't think we have the appetite for that change, but if you want to submit a PR, sim0n00ps can consider that. I'll reopen and flag as a feature request, but absent a PR, I think it's unlikely to happen.
